tests/unit/aiplatform/test_custom_job.py

```diff
@@ -260,24 +241,25 @@ def create_custom_job_mock_with_enable_web_access():
 
 
 @pytest.fixture
-def create_custom_job_mock_fail():
+def create_custom_job_mock_with_tensorboard():
     with mock.patch.object(
         job_service_client.JobServiceClient, "create_custom_job"
     ) as create_custom_job_mock:
-        create_custom_job_mock.side_effect = RuntimeError("Mock fail")
+        custom_job_proto = _get_custom_job_proto(
+            name=_TEST_CUSTOM_JOB_NAME,
+            state=gca_job_state_compat.JobState.JOB_STATE_PENDING,
+        )
+        custom_job_proto.job_spec.tensorboard = _TEST_TENSORBOARD_NAME
+        create_custom_job_mock.return_value = custom_job_proto
         yield create_custom_job_mock
 
 
 @pytest.fixture
-def create_custom_job_v1beta1_mock():
+def create_custom_job_mock_fail():
     with mock.patch.object(
-        job_service_client_v1beta1.JobServiceClient, "create_custom_job"
+        job_service_client.JobServiceClient, "create_custom_job"
     ) as create_custom_job_mock:
-        create_custom_job_mock.return_value = _get_custom_job_proto(
-            name=_TEST_CUSTOM_JOB_NAME,
-            state=gca_job_state_compat.JobState.JOB_STATE_PENDING,
-            version="v1beta1",
-        )
+        create_custom_job_mock.side_effect = RuntimeError("Mock fail")
```

Comment:
What is this "Mock fail" side effect for?

Reply:
This seems to be git diff surfacing the change in a confusing way.

`create_custom_job_mock_fail` hasn't changed, reference line 263 on the previous version. It's used to test that we handle failure from the service.

`create_custom_job_v1beta1_mock` was changed to `create_custom_job_mock_with_tensorboard` above.